shared/events.d.ts

```diff
@@ -127,6 +127,10 @@ interface GlobalEventNameMap {
 	'SliderValueChanged':					(sourceID: string, value: float) => void,
 	'SpinnerValueChanged':					(sourceID: string, value: float) => void,
 	'ObserverTargetChanged':				(entIndex: int32) => void,
+	'MapLoaded':							(map_name: string, is_background: boolean) => void,
+	'MapUnloaded':							() => void,
+	'Cancelled':							(sourceID: string, source: PanelEventSource) => void,
+	'PopulateLoadingScreen':				(map_name: string) => void,
```

Comment:
Nit: it baaaarely matters but the variable names in the callback should be camel cased. Some linters may yell at you to use the same name in the callback as the type.

Reply:
Alright, I've fixed this